bgpd/bgp_route.c

```diff
@@ -4588,18 +4628,194 @@ static void bgp_soft_reconfig_table(struct peer *peer, afi_t afi, safi_t safi,
 		}
 }
 
+/* Do soft reconfig table per bgp table.
+ * Walk on SOFT_RECONFIG_TASK_MAX_PREFIX bgp_dest,
+ * when BGP_NODE_SOFT_RECONFIG is set,
+ * reconfig bgp_dest for list of table->soft_reconfig_peers peers.
+ * Schedule a new thread to continue the job.
+ * Without splitting the full job into several part,
+ * vtysh waits for the job to finish before responding to a BGP command
+ */
+static int bgp_soft_reconfig_table_task(struct thread *thread)
```

Comment:
This function always returns 0, remove the return value

Reply:
In lib/thread.c it is expected an int return value.

```
struct thread *funcname_thread_add_timer(struct thread_master *m,
					 int (*func)(struct thread *),
					 void *arg, long timer,
					 struct thread **t_ptr, debugargdef)
```